src/js/models/StopDetails.js

```diff
@@ -40,12 +34,8 @@ StopDetails.prototype.fetch = function() {
     function retryAtMost(maxRetries) {
         requests.get(yqlURL, params)
             .then(this.parseResponse.bind(this))
-            .tap(function(Services) {
-                var tripCollections = Services.map(function(Service) {
-                    return new TripCollection(this.stopID(), Service);
-                }.bind(this));
-
-                this.tripCollections(tripCollections);
+            .tap(function(Runs) {
+                this.tripCollection = new TripCollection(this.stopID(), Runs);
```

Comment:
I agree, I'm thinking `retryAtMost` should be an extension of `requests`, with `maxRetries` as an option passed to `requests.*()` calls.